cmake/ext_ortlib.cmake

```diff
@@ -62,6 +64,18 @@ else()
     set(ONNXRUNTIME_INCLUDE_DIR ${onnxruntime_SOURCE_DIR}/headers)
     set(ONNXRUNTIME_LIB_DIR ${onnxruntime_SOURCE_DIR}/jni/${ANDROID_ABI})
     message(STATUS "Android onnxruntime inc=${ONNXRUNTIME_INCLUDE_DIR} lib=${ONNXRUNTIME_LIB_DIR}")
+  elseif(IOS)
+    set(ONNXRUNTIME_INCLUDE_DIR ${onnxruntime_SOURCE_DIR}/Headers)
+    if(CMAKE_OSX_SYSROOT STREQUAL "iphoneos")
+      set(ONNXRUNTIME_LIB_DIR ${onnxruntime_SOURCE_DIR}/onnxruntime.xcframework/ios-arm64)
+    elseif(CMAKE_OSX_SYSROOT STREQUAL "iphonesimulator")
+      set(ONNXRUNTIME_LIB_DIR ${onnxruntime_SOURCE_DIR}/onnxruntime.xcframework/ios-arm64_x86_64-simulator)
+    else()
+      message(FATAL_ERROR "Unsupported CMAKE_OSX_SYSROOT value: ${CMAKE_OSX_SYSROOT}")
+    endif()
+    # hack: set ONNXRUNTIME directly to the static library in the framework
+    # couldn't figure out how to get find_library to work with a framework and it doesn't support xcframeworks yet
```

Comment:
try again, maybe it will work after removing the CMAKE_FIND_FRAMEWORK = NEVER assignment